ionic-app/store/actions.js

```diff
@@ -0,0 +1,32 @@
+import Store from '.';
+
+export const setMenuOpen = open => {
+  Store.update(s => {
+    s.menuOpen = open;
+  });
+};
+
+export const setNotificationsOpen = open => {
+  Store.update(s => {
+    s.notificationsOpen = open;
+  });
+};
+
+export const setSettings = settings => {
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/toost-net/toost-cloud/pullRequest?prid=8079923)
```suggestion
export const setSettings = (settings) => {
```